dev/release/post-08-docs.sh

```diff
@@ -83,6 +83,8 @@ git checkout docs/c_glib/index.html
 if [ "$is_major_release" = "yes" ] ; then
   mv docs_temp docs/${previous_version}
 fi
+# Correct the stable version in versionwarnings.js
+sed -i -e "s/${previous_version}/${version}/g" docs/versionwarning.js
```

Comment:
BTW, we have https://arrow.apache.org/docs/_static/versions.json . Can we use it instead of embedding the latest stable version into https://github.com/AlenkaF/arrow-site/blob/73fb93f1f2254efd11b83e8de8b020d085d009e7/docs/versionwarning.js#L4 ?

Reply:
Thank you Kou for all of the suggestions! Will make changes to the PR today.

Reply:
Changes to the JS file (using `/docs/_static/versions.json` to get the stable version) can be found here:
https://github.com/apache/arrow-site/pull/275/commits/7fab4a9ca3751d45da0861279c558e79e5554667